features/chwd: Use global autoconfig and sync commands to latest version

1Naim opened the pull request.

1Naim committed:
features/chwd: Update commands to reflect latest chwd

Signed-off-by: Eric Naim <dnaim@cachyos.org>

vnepogodin commented on src/content/docs/features/chwd.md:
```diff
@@ -17,15 +17,17 @@ to use it post-install.
 We can get class IDs by running `lspci`
```
need to replace `lspci` here then

1Naim replied:
Oops.

vnepogodin reviewed (approved):
LGTM

ventureoo commented on src/content/docs/features/chwd.md:
```diff
@@ -14,18 +14,20 @@ to use it post-install.
 ### Auto Configuration via Class IDs
 
-We can get class IDs by running `lspci`
+We can get class IDs by running the following command
```
What is the point to run chwd with explicit set class id instead of autoconfiguring all devices with any class id? I mean, the average user just wants all their drivers to be installed properly.

1Naim replied:
I wasn't aware that -a runs without accepting any additional arguments. Just noticed that theres a bracket wrapped around `<classid>`. I assume that implies explicit class id is optional. I'll update accordingly then

1Naim committed:
features/chwd: Auto configure without explicitly passing a class ID

chwd supports auto configuring without needing to explicitly pass a class ID.
Due to this and the ability to install a specific profile, it seems a bit redundant to document this as well,
as some might categorize it as overinformation. The removed passage might be added back in the future, but for now
the current information provided is sufficient.

Signed-off-by: Eric Naim <dnaim@cachyos.org>

ventureoo reviewed (approved):
LGTM
